src/scan/postgres_seq_scan.cpp

```diff
@@ -68,25 +68,23 @@ PostgresSeqScanFunction::PostgresSeqScanFunction()
 }
 
 duckdb::unique_ptr<duckdb::GlobalTableFunctionState>
-PostgresSeqScanFunction::PostgresSeqScanInitGlobal(duckdb::ClientContext &context,
-                                                   duckdb::TableFunctionInitInput &input) {
+PostgresSeqScanFunction::PostgresSeqScanInitGlobal(duckdb::ClientContext &, duckdb::TableFunctionInitInput &input) {
 	auto &bind_data = input.bind_data->CastNoConst<PostgresSeqScanFunctionData>();
 	auto global_state = duckdb::make_uniq<PostgresSeqScanGlobalState>(bind_data.m_rel, input);
 	global_state->m_global_state->m_snapshot = bind_data.m_snapshot;
-	return std::move(global_state);
```

Comment:
Is this one not needed anymore?

Reply:
It was redundant (prevents some compiler optimizations then), and caught by a compiler warning.